### DISCUSSION TITLE
Chat or community?

### DISCUSSION THREAD

**Discussion by brandostrong** [voicepaw/so-vits-svc-fork#623] [General]
Hi, I'd like to start contributing to the code, is there an area like discord or irc where the contributors communicate in realtime?  I have some questions on how to get a dev environment set up, and how best to test/debug. I'd like to work on adding more customization during the preprocessing steps, and improving how models are organized and saved.

**Comment by 34j** [comment]
Contributing: Contributing.md
Discord: #312 (abandoned)
